db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/MethodCallNode.java

```diff
@@ -1318,4 +1354,37 @@ public void setHashableJoinColumnReference(ColumnReference cr) {
         }
 	}
 
+	/**
+	 * Add the one single parameter
+	 *
+	 * @param newParm		The new parameter to be appended to methodParms
+	 *
+	 * @exception StandardException		Thrown on error
+	 */
+	public void addOneParm(QueryTreeNode newParm) throws StandardException
+	{
+		JavaValueNode[] newMethodParms = new JavaValueNode[methodParms.length + 1];
+
+		QueryTreeNode qt = (QueryTreeNode) newParm;
+
+		/*
+		 ** Since we need the parameter to be in Java domain format, put a
+		 ** SQLToJavaValueNode on top of the parameter node if it is a
+		 ** SQLValueNode. But if the parameter is already in Java domain
+		 ** format, then we don't need to do anything.
+		 */
+		if ( ! (qt instanceof JavaValueNode))
```

Comment:
Comment says we expect a JavaValueNode or SQLValue node here.  We could throw an exception here if qt is neither of those nodes.